ChattoAdditions/Source/Chat Items/CompoundMessage/Content/MessageContentFactoryProtocol.swift

```diff
@@ -41,20 +43,26 @@ public protocol MessageContentFactoryProtocol {
     func canCreateMessageModule(forModel model: Model) -> Bool
     func createMessageModule(forModel model: Model) -> MessageContentModule
     func createLayoutProvider(forModel model: Model) -> MessageManualLayoutProviderProtocol
+    func createMenuPresenter(forModel model: Model) -> ChatItemMenuPresenterProtocol?
 }
 
 public final class AnyMessageContentFactory<Model>: MessageContentFactoryProtocol {
 
     private let _canCreateMessageModule: (Model) -> Bool
     private let _createMessageModule: (Model) -> MessageContentModule
     private let _createLayoutProvider: (Model) -> MessageManualLayoutProviderProtocol
+    private let _createMenuPresenter: (Model) -> ChatItemMenuPresenterProtocol?
 
     public init<U: MessageContentFactoryProtocol>(_ base: U) where U.Model == Model {
+        self.identifier = "\(type(of: base as Any))"
```

Comment:
Can we add identifier to the protocol and move this implementation to the default implementation of the protocol?

Reply:
Done.